1159/en.md

```diff
@@ -0,0 +1,82 @@
+
+# [LOJ 1159 Batman](https://lightoj.com/problem/batman)
+
+## Solution Approach:
+In this problem given 3 string and need to find the length of the longest common subsequence 
+[LCS](https://www.programiz.com/dsa/longest-common-subsequence) of these three strings.
+
+Its not necessary that the characters have to be next to each other 
+in the original strings.
```

Comment:
Confusing line. Unnecessary. I would remove it, since sub-sequence by itself means that, but more definitively.